docs/source/running.rst

```diff
@@ -101,3 +101,28 @@ While still inside the virtual environment, run::
                  --deployment-dir <contract-deployment-dir> \
                  --source-chain source \
                  --target-chain target
+
+Updating an agent
+-----------------
+
+To describe how to update an agent, it is worth to have a look on Beamer's versioning scheme. As described in
+branching strategy, each major version describes a different mainnet deployment thus a different set of contracts.
+Please note, that if you update your agent to a new major version, it will run on different contracts. Updating to a
+#new major version requires different steps for you to safely transition to a new major contract version.
+Any minor version upgrade indicates a new agent release (i.e. 1.X.0 -> 1.Y.0).
```

Comment:
Thinking more about this, I would maybe provide more reassurance here, by saying e.g.

```
A minor (e.g. 1.X.0 -> 1.Y.0) or a patch (e.g. 1.2.X -> 1.2.Y) version upgrade typically brings with it a
set of fixes and it is recommended to switch to a newer version sooner rather than later.
The difference between the minor and patch version updates is that the former may bring a
change in command line options, configuration file settings or similar things where some user
attention may be required, while the patch version update should be completely painless.
```